src/OpenTelemetry/Trace/TracerProviderBuilderSdk.cs

```diff
@@ -129,55 +129,32 @@ internal TracerProviderBuilder AddProcessor(BaseProcessor<Activity> processor)
             return this;
         }
 
-        internal TracerProvider Build()
-        {
-            return new TracerProviderSdk(
-                this.resourceBuilder.Build(),
-                this.sources,
-                this.diagnosticSourceInstrumentationFactories,
-                this.instrumentationFactories,
-                this.sampler,
-                this.processors);
-        }
-
         /// <summary>
-        /// Adds a DiagnosticSource based instrumentation.
-        /// This is required for libraries which is already instrumented with
-        /// DiagnosticSource and Activity, without using ActivitySource.
+        /// Adds the OperationName of an activity created by DiagnosticSource instrumentation to the provider.
```

Comment:
its misleading to say "created by DiagnosticSource instrumentation".

Reply:
I have updated the description for this to match the public extension method's description.